### ISSUE TITLE
What is the purpose of WPSRequest.identifiers?

### ISSUE THREAD

**Issue by cehbrecht** [geopython/pywps#331]
# Description

In the `WPSRequest` class we have the members `identifier` and `identifiers`:
https://github.com/geopython/pywps/blob/34b06f4355cce6eed7d057d17c76b001b4daf0c0/pywps/app/WPSRequest.py#L40

Looks like `identifiers` is a left over and a potential bug. 

# Steps to Reproduce

# Additional Information

See also PR  #330 and `WPSRequest` tests:
https://github.com/geopython/pywps/blob/91da95ef846e4d9bce7cfd8016af467d2acfd1cb/tests/test_wpsrequest.py#L33

**Comment by cehbrecht** [comment]
The `identifier` member was added by PR #312 by @elemoine.

**Comment by ldesousa** [comment]
Yep, looks fishy. `self.identifier` is only used in the JSON serialisation. Will take a closer look over the weekend.

**Comment by elemoine** [comment]
> The identifier member was added by PR #312 by @elemoine.

That is not quite right :) `WPSRequest.identifier` was already there, but it was not serialized by the `json` property. I can provide more details if needed.

**Comment by cehbrecht** [comment]
@elemoine 
Some light on that code part would be nice :) 

It didn't go through the code yet but I have the feeling that there should be only `WPSRequest.identifier`. So we need to refactor and fix the code to get rid of the extra `identifierS`.
